org.eclipse.winery.repository.ui/src/app/section/section.component.html

```diff
@@ -78,16 +78,31 @@
                     </div>
                 </div>
             </div>
-            <winery-namespace-selector
-                name="namespace"
-                required
-                [(ngModel)]="newComponentNamespace"
-                [isRequired]="true">
-            </winery-namespace-selector>
-            <div class="form-group" *ngIf="types">
-                <label for="typeSelect" class="control-label">Type</label>
-                <ng-select id="typeSelect" [items]="types" (selected)="typeSelected($event)"
-                           [active]="[newComponentSelectedType]"></ng-select>
+            <div>
+                <winery-namespace-selector
+                    name="namespace"
+                    required
+                    [(ngModel)]="newComponentNamespace"
+                    [isRequired]="true">
+                </winery-namespace-selector>
+                <div class="form-group" *ngIf="types">
+                    <label for="typeSelect" class="control-label">Type</label>
+                    <ng-select id="typeSelect" [items]="types" (selected)="typeSelected($event)"
+                               [active]="[newComponentSelectedType]"></ng-select>
+                </div>
+            </div>
+
+            <label for="initNamespace" class="control-label">Template: Start of Namespace</label>
```

Comment:
I would prefer something like `Template: Base Namespace`. I think this is more precise than `Start of Namespace`

@koppor what do you think?

Reply:
`Start of namespace URI - used for automatic namespace URI generation`

"Base" has the correlation to inheritance and there is no inheritance for namespaces.